polyfill/README.md

```diff
@@ -9,13 +9,13 @@ To embed the Sign-in Widget via CDN, include links to the JS and CSS files in yo
 
 ```html
 <!-- Polyfill for older browsers -->
-<script src="https://global.oktacdn.com/okta-signin-widget/7.19.1/js/okta-sign-in.polyfill.min.js" type="text/javascript" integrity="sha384-QzQIGwIndxyBdHRQOwgjmQJLod6LRMchZyYg7RUq8FUECvPvreqauQhkU2FF9EGD" crossorigin="anonymous"></script>
+<script src="https://global.oktacdn.com/okta-signin-widget/7.19.2/js/okta-sign-in.polyfill.min.js" type="text/javascript" integrity="sha384-QzQIGwIndxyBdHRQOwgjmQJLod6LRMchZyYg7RUq8FUECvPvreqauQhkU2FF9EGD" crossorigin="anonymous"></script>
 
 <!-- Widget bundle for Okta Identity Engine -->
-<script src="https://global.oktacdn.com/okta-signin-widget/7.19.1/js/okta-sign-in.oie.min.js" type="text/javascript" integrity="sha384-dsJo3gNsmsiwWA84RCi8/8QDZuUuO7qakY04G8EpErXcQStprNhpFUNUOo9OQcLw" crossorigin="anonymous"></script>
+<script src="https://global.oktacdn.com/okta-signin-widget/7.19.2/js/okta-sign-in.oie.min.js" type="text/javascript" integrity="sha384-nnD0YVNPUCj8Y/E/9yQaDjRZCHz8D3sDi87m5TkMb0P2HLvjNgnqrYC+uFzq7DVd" crossorigin="anonymous"></script>
 
 <!-- CSS for widget -->
-<link href="https://global.oktacdn.com/okta-signin-widget/7.19.1/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" integrity="sha384-FL6VsXYuwuq1Zo5lnWVgcOMWSo1JuvDue2bQ66/TdWOTwPEU9OgvF4Ks8fNnaiHd" crossorigin="anonymous" />
+<link href="https://global.oktacdn.com/okta-signin-widget/7.19.2/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" integrity="sha384-4NICyd+flTmlD7KwjKUhd8sZF67pwe/qjqM7A97rp5Lay0KApG081cyie7+5+w0M" crossorigin="anonymous" />
```

Comment:
@denysoblohin-okta are we expecting a change to the `okta-sign-in.min.css` bundle?

Reply:
Doesn't feel like this should change?